bottles/frontend/ui/dialog-env-vars.blp

```diff
@@ -1,40 +1,31 @@
 using Gtk 4.0;
 using Adw 1;
 
-template $EnvVarsDialog: Adw.Window {
-  modal: true;
-  default-width: 500;
-  default-height: 500;
-
-  ShortcutController {
-    Shortcut {
-      trigger: "Escape";
-      action: "action(window.close)";
-    }
-  }
+template $EnvVarsDialog: Adw.Dialog {
```

Comment:
I think it'd be best to change the name of the dialog to `BottlesEnvironmentVariablesDialog`. It's definitely long, but I think avoid abbreviations helps the readability. Prefixing with `Bottles` is also important because libadwaita and GTK follow the same convention.

Reply:
Those libs use their namespaces because they are shared across many projects, but I don't think we need to do that in client code, it's all obviously Bottles, the prefix is unnecessary IMO. I used just `EnvironmentVariablesDialog` like the rest of our classes, I think that's better.

Reply:
Fair enough. I personally prefer the namespace for consistency reasons, but I guess I will have to change it back to how it was everywhere else